WordPress/src/test/java/org/wordpress/android/util/SiteUtilsTest.kt

```diff
@@ -45,4 +46,72 @@ class SiteUtilsTest {
         site.url = "https://***.wordpress.com"
         assertFalse(SiteUtils.hasCustomDomain(site))
     }
+
+    @Test
+    fun `checkMinimalJetpackVersion doesnt fail when Jetpack version is false`() {
+        val site = SiteModel()
+        site.jetpackVersion = "false"
+
+        val hasMinimalJetpackVersion = SiteUtils.checkMinimalJetpackVersion(site, "0")
+
+        assertThat(hasMinimalJetpackVersion).isFalse()
+    }
+
+    @Test
+    fun `checkMinimalJetpackVersion returns true when version higher than input`() {
+        val site = SiteModel()
+        site.jetpackVersion = "5.8"
+        site.origin = 1
+        site.setIsJetpackConnected(true)
+
+        val hasMinimalJetpackVersion = SiteUtils.checkMinimalJetpackVersion(site, "5.6")
+
+        assertThat(hasMinimalJetpackVersion).isTrue()
+    }
+
+    @Test
+    fun `checkMinimalJetpackVersion returns true when version is equal to input`() {
+        val site = SiteModel()
+        site.jetpackVersion = "5.8"
+        site.origin = 1
+        site.setIsJetpackConnected(true)
+
+        val hasMinimalJetpackVersion = SiteUtils.checkMinimalJetpackVersion(site, "5.8")
+
+        assertThat(hasMinimalJetpackVersion).isTrue()
+    }
+
+    @Test
+    fun `checkMinimalJetpackVersion returns false when version is lower than to input`() {
```

Comment:
super minor np; I think the to in `lower than (to) input` could be a copy/paste typo? :)